src/main/java/competition/subsystems/shooterwheel/ShooterWheelSubsystem.java

```diff
@@ -44,7 +47,27 @@ public void setTargetRPM(double speed) {
     }
 
     public double getTargetRPM() {
-        return targetRpmProp.get();
+        if (targetRpmProp.get() == 0) {
```

Comment:
remember, don't compare doubles to 0. You want something closer to:

`if (Math.abs(targetRpmProp.get()) < 50)`